doc/website/getting-started/overview.md

```diff
@@ -183,11 +183,12 @@ The service model of iceoryx is derived from AUTOSAR and is still used in the AP
 
 The following table gives an overview of the different terminologies and the current mapping:
 
-|         | Group   | Instance         | Topic                  |
-|---------|---------|------------------|------------------------|
-| ROS 2   | Type    | Namespace::Topic | -                      |
-| AUTOSAR | Service | Instance         | Event                  |
-| DDS     | -       | -                | /Group/Instance/Topic  |
+|                                                                                   | Group   | Instance         | Topic                  |
```

Comment:
How is this file related to the design decision document? Doesn't it violate our traceability rule when we change files which are unrelated to the issue?!

Reply:
Nope, this is actually the first step before any implementation can start. We need to consolidate the different mappings as ROS 2, rmw_iceoryx and the DDS gateway in the end all use the same technology and I see no reason to treat them differently. The changes so far only update what is currently implemented.